gulpfile.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
Sounds good. Also sounds like I might have another interested party in learning debugging and using VS code?

I think you'll find it to be a really amazing editor.